[2022-05-23 20:15:37,492] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task scheduled__2021-12-31T00:00:00+00:00 [queued]>
[2022-05-23 20:15:37,506] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task scheduled__2021-12-31T00:00:00+00:00 [queued]>
[2022-05-23 20:15:37,512] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:15:37,512] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-05-23 20:15:37,512] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:15:37,576] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): thrid_task> on 2021-12-31 00:00:00+00:00
[2022-05-23 20:15:37,590] {standard_task_runner.py:52} INFO - Started process 2142 to run task
[2022-05-23 20:15:37,604] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'thrid_task', 'scheduled__2021-12-31T00:00:00+00:00', '--job-id', '466', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpod3pvyqh', '--error-file', '/tmp/tmppvvypzdj']
[2022-05-23 20:15:37,605] {standard_task_runner.py:80} INFO - Job 466: Subtask thrid_task
[2022-05-23 20:15:37,700] {task_command.py:369} INFO - Running <TaskInstance: our_first_dag_v5.thrid_task scheduled__2021-12-31T00:00:00+00:00 [running]> on host 061a475e6c14
[2022-05-23 20:15:37,836] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=gerson
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=thrid_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-31T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-31T00:00:00+00:00
[2022-05-23 20:15:37,837] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-23 20:15:37,838] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task3 and will be running after task1 at the same time as task2!']
[2022-05-23 20:15:37,864] {subprocess.py:85} INFO - Output:
[2022-05-23 20:15:37,866] {subprocess.py:92} INFO - hey, I am task3 and will be running after task1 at the same time as task2!
[2022-05-23 20:15:37,866] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-23 20:15:37,938] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=thrid_task, execution_date=20211231T000000, start_date=20220523T201537, end_date=20220523T201537
[2022-05-23 20:15:37,986] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-23 20:15:38,045] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
